### ISSUE TITLE
Require the WRC to publicly announce “unwritten regs”

### ISSUE THREAD

**Issue by UnderwaterCuber** [thewca/worldcubeassociation.org#4752]
Not 100% sure if this is the right place for this, if it isn’t the please direct me to the right place :P 

Recently I have seen two rulings about the regs made by the WRC, one was never announced in a public way and neither are easily findable. The first was allowing speedcubeshops PVC coated cubes to be used, the cubes have a transparent coating which has clear stickers embedded, this isn’t allowed by 3h2a. The other was allowing delegates to not grade competitors solutions if the competitor submitted intentionally hard to execute solutions or submitted am intentionally poor solution (excessive use of wide moves or rotations, doing things like U U U U2 U’, getting 69 moves on ever solve, etc.) by applying A1c to FMC (article A doesn’t apply to FMC). While I think both of these are good things to do I don’t think the way that they did it was appropriate or upholds the spirit of the WCA. Due to neither of these never being announced it would be very possible for people to be receiving differing rulings for the same things and for competitors to not understand what is and isn’t legal. Competitor A in location A may not be be allowed to use their PVC coated cube or may have their average DNFd after the delegate discovers them using it, competitor B in location B may not bring their PVC coated cube to the competition at all since they read the regs and know it’s illegal, competitor C in location C uses their PVC coated cube and has no repercussions. In the case of FMC a competitor goofing around after they find a good solution with plenty of time to spare may transform their solution to all wide moves knowing it’s legal thinking others may find it humorous, they receive a DNF while on the other side of the world another competitors solution uses nothing but rotations and Dw moves with no repercussions. In a case like FMC it’s also unfair to the graders seeing how one had to spend the time to grade a ridiculous solution while the other rejected it immediately. 

The long story short this creates unfairness and confusion and needs to stop. Whether they change the regs directly or have somewhere to publicly announce it the way they are currently doing it is unacceptable.

**Comment by lgarron** [comment]
We currently use the Incidents log for this: https://www.worldcubeassociation.org/incidents

Regarding these particular topics:

## PVC coated cubes

This was not entered in the Incidents log yet, and we hadn't received any public inquiries about it. Since you brought it up here, I've gone ahead and published an entry: https://www.worldcubeassociation.org/incidents/42

In the past, the WRC has received criticism for being slow to allow puzzle modifications that many think don't provide an unfair advantage. Do you think it would have been any better to disallow these puzzles for now and allow them in a future Regs update?

## A1c and Fewest Moves

A1c was originally designed to apply to Fewest Moves: https://github.com/thewca/wca-regulations/issues/225
This WRC has correspondingly allowed the use of A1c for Fewest Moves since its introduction.
Since this formatting issue was brought up recently, we have a pending change for 2020 to make it clear: https://github.com/thewca/wca-regulations/pull/868

In any case, we can't solve the fundamental problem that Delegate discretion will vary from competition to competition.

--------

In my experience, Delegates used to agree on *many* unwritten rules and the Regulations had many consistency issues that "technically" contradicted common practice. Since the introduction of the WRC, I believe we've gotten a lot better at making the wording of the Regulations match the intent, to the point that the A1c issue looks like a sore thumb in retrospect.

Based on talking to the Board and Delegates, I don't think it's a good idea to make major changes to the Regulations more than once (maybe twice) a year, so the Incidents log is the place for "unclarified" and "unwritten" Regs.

I think it's good to call us out if we're not publishing important clarifications, but are you seeing issues beyond what you have mentioned here?
